img/media-types.dot

```diff
@@ -1,13 +1,20 @@
 digraph G {
   {
     imageIndex [shape=note, label="Image Index\n<<optional>>\napplication/vnd.oci.image.index.v1+json"]
-    manifest [shape=note, label="Image manifest\napplication/vnd.oci.image.manifest.v1+json"]
-    config [shape=note, label="Image JSON\napplication/vnd.oci.image.config.v1+json"]
+    {
+      rank=same
+      manifest [shape=note, label="Image manifest\napplication/vnd.oci.image.manifest.v1+json"]
+      artifact [shape=note, label="Artifact Manifest\napplication/vnd.oci.artifact.manifest.v1+json"]
+    }
+    config [shape=note, label="Image config JSON\napplication/vnd.oci.image.config.v1+json"]
     layer [shape=note, label="Layer tar archive\napplication/vnd.oci.image.layer.v1.tar\napplication/vnd.oci.image.layer.v1.tar+gzip\napplication/vnd.oci.image.layer.nondistributable.v1.tar\napplication/vnd.oci.image.layer.nondistributable.v1.tar+gzip"]
   }
 
   imageIndex -> imageIndex [label="1..*"]
   imageIndex -> manifest [label="1..*"]
   manifest -> config [label="1..1"]
   manifest -> layer [label="1..*"]
+  artifact -> manifest [label="1..1"] [constraint = false];
```

Comment:
If this is capturing the "subject" field, then we also need two more self references like there is for `imageIndex`, and the reference is also optional, so `0..1`:

```suggestion
  artifact -> manifest [label="0..1"] [constraint = false];
  artifact -> artifact [label="0..1"];
  manifest -> manifest [label="0..1"];
```

I'm not sure how to capture that it's `0..1` for multiple targets and you can't have a manifest with both a subject of another manifest and an artifact, but that may be more detail than needed.